### PR TITLE
Feature: CoderHeroes Theme

### PR DESCRIPTION
**Pull request by waylonturbes**
Co-authored-by: priscila monteiro <pririmonteiro@me.com>
Co-authored-by: moduped <daniemu@mail.uc.edu>
Co-authored-by: Nathan Cheney <natecheney@gmail.com>

## Description

- Created a theme file called `coderHeroesTheme.js`
- Added all color-styles and text-styles from Figma to `coderHeroesTheme.js`
- Added `coderHeroesTheme.js` to less loader options `globalVars`, which allows all less files to access the theme's variables
- Renamed `theme-overrides.js` to `themeOverrides.js`
- Reverted `themeOverrides.js` to Ant Designs default
- Updated `login.less` to resolve any errors caused by renaming previous global less variables
- Minor tweaks to `login.less` so that it is more accessible (button foreground color wasn't passing WCAG AA)

**Additional Changes:**
- Fixed re-render error for Okta sign-in widget
- Resolved `useEffect` empty dependency compile warning in `LoginContainer.js`

## Video

https://youtu.be/6itszcuiqqk

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Comment by waylonturbes** [comment]
[Add color styles and fonts from Figma to our theme](https://trello.com/c/SXXR31ZS/191-add-color-styles-and-fonts-from-figma-to-our-theme)

**Review by Cheyenneb96 — APPROVED** [review]
looks good this will definitely make thanks easier moving forward

File: src/components/pages/Login/LoginContainer.js [hunk 1]
```diff
@@ -7,9 +7,8 @@ import '../../../styles/login.less';
 import { config } from '../../../utils/oktaConfig';
 
 const LoginContainer = () => {
-  let history = useHistory();
-
-  useEffect(() => {
+  const history = useHistory();
+  const loadWidget = useCallback(() => {
```
Comment (katherineyevsukov) [inline src/components/pages/Login/LoginContainer.js:11]: Nice job fixing the linter warning by moving refactoring as a useCallback! Great solution.

**Review by katherineyevsukov — APPROVED** [review]
Wonderful job with implementing the themes. This will be so helpful for devs going forward and will provide consistency on the front end. Also, really important you made sure accessibility was up to par. You all are crushing it!

**Review by mark-escosura — APPROVED** [review]

**Review by natecheney-dev — APPROVED** [review]

**Comment by ModupeD** [comment]
The color styles are very important, it will help the website look nicer and more uniform.

**Pull request merged**